app/views/hyrax/dashboard/collections/_subcollection_list.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @subcollection_docs.nil? || @subcollection_docs.empty? %>
+  <div class="alert alert-warning" role="alert"><%= t('hyrax.collections.show.no_visible_subcollections') %></div>
+<% else %>
+  <div class="media-body">
+    <h4 class="media-heading">
+      <% @subcollection_docs.each do |document| %>
+        <ul>
+         <li>
+            <% id = document.id %>
+            <%= link_to document.title_or_label, [hyrax, :dashboard, document], id: "src_copy_link_#{id}" %> (<%= "@TODO" %>)
```

Comment:
Would `<%= collection_presenter.total_viewable_collections %>` work here?  It appears that for works, the count is coming from the search builder that blacklight is using to show the works.  I wonder if that and `collection_presenter.total_viewable_works` would ever be different.